Kernel/Arch/aarch64/Registers.h

```diff
@@ -697,6 +697,28 @@ struct alignas(u64) SCTLR_EL1 {
 };
```

Comment:
Commit message again (2bc99853ea6ea32f353890fb9d36e9acef9fc4c9), and I also have a commit for this in #17258 (8c7c20f153a5b79c09bbdae868faf24f4adc3f14) where I approached it differently, as we already restore `tpidr_el0` in RegisterState. I'm not sure which approach is better.

Reply:
i must say iʼve liked this central point here for all the registers… and snippets also in one place.

still iʼll follow your lead here! your call?

Reply:
>  must say iʼve liked this central point here for all the registers… and snippets also in one place.

Yes I guess this is fine too, however that means we don't need to store tpidr_el0 in the RegisterState.h, but no need to add that to this PR.

Still the commit message isn't very clear, it should at least mention the TPIDR_EL0  register.